node-repository/src/test/java/com/yahoo/vespa/hosted/provision/autoscale/MetricsV2MetricsFetcherTest.java

```diff
@@ -161,6 +161,14 @@ public void close() { }
             "            },\n" +
             "            {\n" +
             "              \"values\": {\n" +
+            "                \"content.proton.resource_usage.memory.average\": 35,\n" +
```

Comment:
These metrics values are between 0 and 1

Reply:
thanks!